vllm/entrypoints/openai/cli_args.py

```diff
@@ -112,11 +112,14 @@ def make_arg_parser(parser: FlexibleArgumentParser) -> FlexibleArgumentParser:
                         type=json.loads,
                         default=["*"],
                         help="Allowed headers.")
-    parser.add_argument("--api-key",
-                        type=optional_type(str),
-                        default=None,
-                        help="If provided, the server will require this key "
-                        "to be presented in the header.")
+    parser.add_argument(
+        "--api-key",
+        type=optional_type(str),
+        default=None,
+        nargs="+",
```

Comment:
When merge from main, instead of manually adding `nargs` simply update the typt hint to `list[str]` for `FrontendArgs.api_key`. Our CLI machinery will handle the rest

Reply:
Hello, thanks, fixed it! Could you review pls?